LDMP/models/datasets_model.py

```diff
@@ -56,7 +58,7 @@ def data(self, index: QModelIndex = QModelIndex(), role: Qt.ItemDataRole = Qt.Di
         item = index.internalPointer()
         if role == Qt.DisplayRole:
             if index.column() == 0:
-                return list(self.rootItem.items())[index.row()].name
+                return list(self.rootItem.items())[index.row()][1]
```

Comment:
Because rootItem is a Datasets e.g.s a dict with key=json and value=Dataset and Dataset is stored as Item (see Datasets.child(...) method
I would simplify:
```suggestion
                return item
```
plus... I would remove line 60 because now unuseful